src/Microsoft.Health.Fhir.SqlServer/Features/Storage/SqlServerFhirDataStore.cs

```diff
@@ -949,6 +959,91 @@ private bool ExistingRawResourceIsEqualToInput(RawResource input, RawResource ex
             }
         }
 
+        private bool ChangesAreOnlyInMetadata(ResourceWrapper inputWrapper, ResourceWrapper existingWrapper)
+        {
+            var inputResource = (Hl7.Fhir.Model.Resource)_resourceDeserializer.Deserialize(inputWrapper).ResourceInstance;
+            var existingResource = (Hl7.Fhir.Model.Resource)_resourceDeserializer.Deserialize(existingWrapper).ResourceInstance;
+
+            var inputMeta = inputResource.Meta;
+            var existingMeta = existingResource.Meta;
+
+            if (inputMeta.Equals(existingMeta))
+            {
+                return false; // Difference is in a non-meta field
+            }
+            else
+            {
+                var inputChildren = inputResource.NamedChildren.GetEnumerator();
```

Comment:
I see we have a method: ExistingRawResourceIsEqualToInput, which replaces some values in the meta field, and then simply compares the two rawresources.  Could we do something similar where we replace the entire meta field from the incoming with the existing and then simply compare the entire resource?

Reply:
@feordin :-) I added my comment above before reading yours...

Reply:
I changed the way this is handled, it now just replaces the meta field with empty.